packages/kamino-sdk/tests/stratCreation.test.ts

```diff
@@ -506,4 +496,80 @@ describe('Kamino strategy creation SDK Tests', () => {
     strategyData = await kamino.getStrategies([newStrategy.publicKey]);
     expect(strategyData[0]?.rebalanceType == Manual.discriminator);
   });
+
+  it.skip('create new custom USDC-USDH percentage strategy on existing whirlpool and open position', async () => {
```

Comment:
why skip? explain in comment/name

Reply:
we skip all of them because it requires a mainnet private key with some SOL to be able to run them and if it is not provided they will just fail